src/main/java/uk/gov/register/views/representations/CsvWriter.java

```diff
@@ -27,6 +28,7 @@ public void writeTo(CsvRepresentationView view, Class<?> type, Type genericType,
         CsvRepresentation csvRepresentation = view.csvRepresentation();
 
         objectMapper.writerFor(csvRepresentation.contentType)
+                .with(JsonGenerator.Feature.IGNORE_UNKNOWN)
```

Comment:
It appears that the problem this line used to address (mapping an unknown field of `key`) no longer exists, so I'll undo this change.